### PR TITLE
Feature session scroll behavior

### PR DESCRIPTION
**Pull request by iam-robin**

### CODE REVIEW

File: src/components/session/SessionItemComponent.tsx [hunk 1]
```diff
@@ -161,20 +256,43 @@ export const SessionItemComponent = (props: SessionItemProps) => {
...
+				<div
+					className={`session__scrollToBottom ${
+						isScrolledToBottom
+							? 'session__scrollToBottom--disabled'
+							: ''
+					}`}
+				>
+					{newMessages > 0 && (
+						<span className="session__unreadCount">
+							{newMessages > 99 ? '99+' : newMessages}
```
Comment (koepferd) [inline src/components/session/SessionItemComponent.tsx]: Please use a translate variable here.

File: src/components/session/session.styles.scss [hunk 2]
```diff
@@ -89,6 +89,44 @@ $sessionHeaderHeight: 85px;
...
+	}
+
+	&__unreadCount {
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		position: absolute;
+		top: -3px;
+		right: -9px;
+		width: 18px;
```
Comment (koepferd) [inline src/components/session/session.styles.scss]: You might use a variable here for lines 117 & 118

**Review by koepferd — CHANGES_REQUESTED** [review]

File: src/components/session/SessionItemComponent.tsx (hunk 1 above)
Comment (iam-robin) [reply src/components/session/SessionItemComponent.tsx]: 👍

File: src/components/session/session.styles.scss (hunk 2 above)
Comment (iam-robin) [reply src/components/session/session.styles.scss]: 👍

**Review by koepferd — APPROVED** [review]

**Pull request merged**